crates/api_models/src/payments.rs

```diff
@@ -536,6 +537,7 @@ pub enum AdditionalPaymentData {
     PayLater {},
     Crypto {},
     BankDebit {},
+    MandatePayment {},
```

Comment:
mandate payment always requires a `mandate_id`. Can we move the mandate id field here?

Reply:
As discussed with @bernard-eugine it's not a good idea to get mandate_id within payment_method_data as we are getting mandate_data outside of it. So we won't adding mandate id within MandatePayment